spec/models/order_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+
+  let(:candidate) {
+    Candidate.create(name: "Donald Trump", party: "Republican", bio: "Was born in the most luxurious crib the world has ever seen", featured: true, last_name: "trump")
+  }
+
+  let(:issue) {
+    Issue.create(topic: "Environment", description: "Set to be a hotly contested issue", featured: true, picture: 'environment')
```

Comment:
Line is too long. [128/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.